### ISSUE TITLE
Terminal freezes on start

### ISSUE THREAD

**Issue by kristijanhusak** [tarruda/zsh-autosuggestions#67]
Hi,

I have an issue for some time now, and I hoped it will be fixed, but it still isn't. Sometimes when i try to open new split window or open a new terminal window, zsh freezes and it won't show the line to enter anything, just a blinking cursor. Like it is waiting for something. When i press `CTRL+C` to stop the process that it is doing, command line shows, but then i get the message below `No such widget 'autosuggest-start'`. It stands there all the time, `clear` command is not removing it. Also the plugin is not working in that situation. Here's the screenshot:

![screenshot_2015-07-13_10-45-03](https://cloud.githubusercontent.com/assets/1782860/8645952/3db6aaf0-294c-11e5-9896-afaa15cbba28.png)

Sometimes it works out after restarting terminal, and sometimes the only solution is to restart the computer. When I remove autosuggestion init part in the zshrc, everything works fine.

Here's some system information:

System: Xubuntu 15.04 64 bit
Terminal: Terminator
Zsh: Latest version

And this I have in my `.zshrc`:
```
#setup zsh-autosuggestions
source ~/.zsh-autosuggestions/autosuggestions.zsh

# Enable autosuggestions automatically
zle-line-init() {
    zle autosuggest-start
}
zle -N zle-line-init

# use ctrl+t to toggle autosuggestions(hopefully this wont be needed as
# zsh-autosuggestions is designed to be unobtrusive)
bindkey '^T' autosuggest-toggle
AUTOSUGGESTION_HIGHLIGHT_COLOR='fg=6'
AUTOSUGGESTION_HIGHLIGHT_CURSOR=0

```

Is there any way to debug this?

**Comment by githubfun** [comment]
same here

**Comment by ericfreese** [comment]
There's a rewrite underway in PR #91. It's available on branch `v0.1.x`. Please try using that branch, and let me know if it solves your problem.
